tsconfig.build.json

```diff
@@ -0,0 +1,35 @@
+{
+  "include": ["src/**/*"],
+  "compilerOptions": {
+    "module": "commonjs",
+    "strict": true,
+    "declaration": true,
+    "noImplicitAny": true,
+    "removeComments": true,
+    "noLib": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "target": "es2018",
+    "sourceMap": true,
+    "lib": ["es2018", "dom"],
+    "moduleResolution": "node",
+    "outDir": "./dist"
+  },
+  "exclude": [
+    "../../node_modules", "dist", "**/__mocks__/**"],
+  "typedocOptions": {
+    "mode": "modules",
+    "out": "website/docs/api",
```

Comment:
It would make sense that each module has its own API doc. They will hopefully be very similar (at least the user-facing modules, that is all but the `-core`). I guess the best approach would be that each module targets a specific folder, and a top-level command triggers individual module scripts.

Reply:
Ok - done (each sub-package provides it's own 'out' folder).